WordPress/gutenberg
# Add Tip icon to library and use in the Tip component

## Description
This simple PR furthers the mission of ensuring all icons used in the Gutenberg UI are actually available in the [icons library](https://github.com/WordPress/gutenberg/tree/trunk/packages/icons) and are not hardcoded. One such icon is the Tip icon (lightbulb), which is used at the bottom of the inserter (see screenshot).

In this PR, the svg code for the icon is moved to the icons library and the Tip component is updated to reflect this change. This issue was originally mentioned here https://github.com/WordPress/gutenberg/pull/35187#issuecomment-929253365 and I just now got to it 😅

## Testing Instructions
1. Open the inserter and make sure the Tip icon looks the same as it did previously.

## Screenshots 
![image](https://user-images.githubusercontent.com/4832319/152037924-2f19cd95-6ae3-4b2b-8260-37081d03fa36.png)


## Types of changes
New feature

## Checklist:
